### PR TITLE
[doc] Update project directory structure in docs

### LINKED ISSUE #9670: [Bug] [Doc] Update Project directory structure
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

* Project directory structure has changed lot, we need to update docs of both English and Chinese versions.
* See https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/architecture/configuration.md

### What you expected to happen

* Already described above

### How to reproduce

* Already described above

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Update project directory structure in docs, see https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/architecture/configuration.md
* This PR closes: #9670 

## Brief change log
* Already described above

### CODE REVIEW

**Comment by EricGao888** [comment]
Still WIP. Since v3.0.0 refactors the project in `microservices` architecture, there are several duplicate scripts and configs across modules. I've updated the project structure in English doc and reviews are appreciated. If it looks fine, I will update the Chinese doc in the next commit. Thx : )

**Comment by zhongjiajie** [comment]
Looking great! will take a look this date later 👍

File: docs/docs/en/architecture/configuration.md [hunk 1]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
-├─ui                                front-end web resources directory
-├─lib                               DS .jar dependencies directory
-├─install.sh                        auto-setup DS services script
+├── LICENSE
+│
+├── NOTICE
+│
+├── licenses                                    directory of licenses
+│
+├── bin                                         directory of DS application commands, configrations scripts
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]: We should avoid using the shortcut for product name in our document
```suggestion
├── bin                                         directory of DolphinScheduler application commands, configrations scripts 
```

Comment (zhongjiajie) [reply docs/docs/en/architecture/configuration.md]: same as other keyword below

File: docs/docs/en/architecture/configuration.md [hunk 2]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+├── LICENSE
+│
+├── NOTICE
+│
+├── licenses                                    directory of licenses
+│
+├── bin                                         directory of DS application commands, configrations scripts 
+│   ├── dolphinscheduler-daemon.sh              script to start or shut down DS application
+│   ├── env                                     directory of scripts to load environment variables
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables [eg: JAVA_HOME,HADOOP_HOME, HIVE_HOME ...]
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   │   ├── dolphinscheduler_env.sh             script to load environment variables [eg: JAVA_HOME,HADOOP_HOME, HIVE_HOME ...], will export them when you start or stop service using script `dolphinscheduler-daemon.sh` 
```

File: docs/docs/en/architecture/configuration.md [hunk 3]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   │   └── install_env.sh                      script to load environment variables for DS installation
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   │   └── install_env.sh                      script to load environment variables for DS installation, will export them when you using script `install.sh` `start-all.sh` `stop-all.sh` `status-all.sh`
```

File: docs/docs/en/architecture/configuration.md [hunk 4]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   ├── install.sh                              script to auto-setup DS services
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   ├── install.sh                              script to auto-setup DS services, use when you deploy with psuedo-cluster mode or cluster mode
```

File: docs/docs/en/architecture/configuration.md [hunk 5]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│
+├── bin                                         directory of DS application commands, configrations scripts 
+│   ├── dolphinscheduler-daemon.sh              script to start or shut down DS application
+│   ├── env                                     directory of scripts to load environment variables
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables [eg: JAVA_HOME,HADOOP_HOME, HIVE_HOME ...]
+│   │   └── install_env.sh                      script to load environment variables for DS installation
+│   ├── install.sh                              script to auto-setup DS services 
+│   ├── remove-zk-node.sh                       script to cleanup ZooKeeper caches 
+│   ├── scp-hosts.sh                            script to copy installation files to target hosts 
+│   ├── start-all.sh                            script to start all DS services
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   ├── start-all.sh                            script to start all DS services, use when you deploy with pseudo-cluster mode or cluster mode
```

File: docs/docs/en/architecture/configuration.md [hunk 6]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   ├── status-all.sh                           script to check the status of all DS services
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   ├── status-all.sh                           script to check the status of all DS services, use when you deploy with pseudo-cluster mode or cluster mode
```

File: docs/docs/en/architecture/configuration.md [hunk 7]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   ├── dolphinscheduler-daemon.sh              script to start or shut down DS application
+│   ├── env                                     directory of scripts to load environment variables
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables [eg: JAVA_HOME,HADOOP_HOME, HIVE_HOME ...]
+│   │   └── install_env.sh                      script to load environment variables for DS installation
+│   ├── install.sh                              script to auto-setup DS services 
+│   ├── remove-zk-node.sh                       script to cleanup ZooKeeper caches 
+│   ├── scp-hosts.sh                            script to copy installation files to target hosts 
+│   ├── start-all.sh                            script to start all DS services
+│   ├── status-all.sh                           script to check the status of all DS services 
+│   └── stop-all.sh                             script to shut down all DS services
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
│   └── stop-all.sh                             script to shut down all DS services, use when you deploy with pseudo-cluster mode or cluster mode
```

File: docs/docs/en/architecture/configuration.md [hunk 8]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+├── api-server                                  directory of DS api-server commands, configrations scripts and libs
+│   ├── bin
+│   │   └── start.sh                            script to start DS api-server
+│   ├── conf
+│   │   ├── application.yaml                    configurations of api-server
+│   │   ├── common.properties                   configurations of common-service like storage, credentials, etc.
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables for api-server
+│   │   └── logback-spring.xml                  configurations of api-service log
+│   ├── libs                                    directory of api-server libs
+│   └── ui
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]: should we also add some description about `ui` component

File: docs/docs/en/architecture/configuration.md [hunk 9]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   ├── bin                                
+│   │   └── start.sh                            script to start DS master-server
+│   ├── conf
+│   │   ├── application.yaml                    configurations of master-server
+│   │   ├── common.properties                   configurations of common-service like storage, credentials, etc.
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables for master-server
+│   │   └── logback-spring.xml                  configurations of master-service log
+│   └── libs                                    directory of master-server libs
+│
+├── standalone-server                                       directory of DS standalone-server commands, configrations scripts and libs
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
├── standalone-server                          directory of DS standalone-server commands, configrations scripts and libs
```

Comment (zhongjiajie) [reply docs/docs/en/architecture/configuration.md]: indentation

File: docs/docs/en/architecture/configuration.md [hunk 10]
```diff
@@ -15,44 +15,87 @@ This document only describes DolphinScheduler configurations and other topics ar
...
+│   ├── conf
+│   │   ├── application.yaml                    configurations of standalone-server
+│   │   ├── common.properties                   configurations of common-service like storage, credentials, etc.
+│   │   ├── dolphinscheduler_env.sh             script to load environment variables for standalone-server
+│   │   ├── logback-spring.xml                  configurations of standalone-service log
+│   │   └── sql                                 .sql files to create or upgrade DS metadata
+│   ├── libs                                    directory of standalone-server libs
+│   └── ui
+│       
+├── tools                                                       directory of DS metadata tools commands, configrations scripts and libs
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/configuration.md]: indentation here too

File: docs/docs/en/architecture/configuration.md (hunk 1 above)
Comment (EricGao888) [reply docs/docs/en/architecture/configuration.md]: Sure, seems we also need to delete https://github.com/apache/dolphinscheduler/blob/243ee8ba70d50c7d4b251fd43a4c150606014b8b/docs/docs/en/architecture/configuration.md?plain=1#L15

File: docs/docs/en/architecture/configuration.md (hunk 2 above)
Comment (EricGao888) [reply docs/docs/en/architecture/configuration.md]: Thx, will fix it in the next commit.

File: docs/docs/en/architecture/configuration.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/docs/en/architecture/configuration.md]: I never found it before, we could use it only if we have this sentence in the beginning of our document, mean in https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/about/introduction.html

**Comment by zhongjiajie** [comment]
@EricGao888  Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week

**Comment by EricGao888** [comment]
> @EricGao888 Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week

Yes, I'm working on it.

**Comment by EricGao888** [comment]
> @EricGao888 Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week

@zhongjiajie Comments resolved, but need to double check the description for ui folders under standalone-server and api-server. Will update the Chinese doc in next commit if it looks good to you.

**Comment by zhongjiajie** [comment]
> > @EricGao888 Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week
> 
> @zhongjiajie Comments resolved, but need to double check the description for ui folders under standalone-server and api-server. Will update the Chinese doc in next commit if it looks good to you.

LGTM, please also change the ZH docs, thanks

**Comment by EricGao888** [comment]
> > > @EricGao888 Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week
> 
> > 
> 
> > @zhongjiajie Comments resolved, but need to double check the description for ui folders under standalone-server and api-server. Will update the Chinese doc in next commit if it looks good to you.
> 
> 
> 
> LGTM, please also change the ZH docs, thanks

Sure, thx

**Comment by EricGao888** [comment]
> > > @EricGao888 Do you have time to complete this PR recently? we are going to release 3.0.0-beta-1 next week
> > 
> > 
> > @zhongjiajie Comments resolved, but need to double check the description for ui folders under standalone-server and api-server. Will update the Chinese doc in next commit if it looks good to you.
> 
> LGTM, please also change the ZH docs, thanks

Done, PTAL Thx : )

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks a lot

**Pull request merged**